antarest/core/tasks/web.py

```diff
@@ -61,6 +63,13 @@ def get_task(
         Returns:
             TaskDTO: Information about the specified task.
         """
+        expected_pattern = "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
```

Comment:
- I think we should change the existing method "sanitize_uuid" to do this instead, and reuse it where sonar is complaining.
- HTTP status should be bad request 400
- We should compile the regex only one in a "global" variable, for performance reason.